scaaml/capture/scope/lecroy/lecroy.py

```diff
@@ -109,10 +163,14 @@ def __exit__(self, exc_type, exc_value, exc_tb) -> None:
 class LeCroyScope(ScopeTemplate):
     """Scope."""
 
-    def __init__(self, samples: int, offset: int, ip_address: str,
-                 trace_channel: LECROY_CHANNEL_NAME_T,
-                 trigger_channel: LECROY_CHANNEL_NAME_T,
-                 communication_timeout: float, trigger_timeout: float):
+    def __init__(
+        self, samples: int, offset: int, ip_address: str,
+        trace_channel: LECROY_CHANNEL_NAME_T,
+        trigger_channel: LECROY_CHANNEL_NAME_T, communication_timeout: float,
+        trigger_timeout: float, scope_setup_commands: List[Dict[str, Any]],
+        communication_class_name: Literal["LeCroyCommunicationVisa",
```

Comment:
This type seems to appear multiple times. Might be worth creating a type alias in `lecroy.types` module for that.